File: 장세훈/Programmers/Level2/예상대진표.java
class 예상대진표 {
        /*
            https://school.programmers.co.kr/learn/courses/30/lessons/12985
         */
        public int solution(int n, int a, int b) {
            int answer = 0;

            while ((a + 1) / 2 != (b + 1) / 2) {
                answer++;
                a = (a + 1) / 2;
                b = (b + 1) / 2;
            }

            return answer;
        }
    }

File: 오우영/Programmers/Level2/예상대진표.js
// 예상 대진표
// https://school.programmers.co.kr/learn/courses/30/lessons/12985

function solution(n, a, b) {
  let round = 0;

  while (a !== b) {
    a = Math.ceil(a / 2);
    b = Math.ceil(b / 2);
    round++;
  }

  return round;
}
